README.md

```diff
@@ -16,3 +38,50 @@ docker build . -t call-home
 # run the image
 docker run -it call-home
 ```
+
+#### docker-compose
+
+Or more conveniently with docker-compose from repository root.
+
+```bash
+# To include new docker image build
+docker-compose up --build
+
+# To use existing docker image
+docker-compose up
+```
+
+To use postgres as your development database:
+
+```bash
+# backend/.env
+...
+...
+DATABASE_URL=postgresql://postgres:password@db:5432/callhome
```

Comment:
This should be in the docker-compose.yml. I think we can make postgres the default for docker compose.

Reply:
moved to docker-compose.yml